docs/built_with.rst

```diff
@@ -0,0 +1,20 @@
+Built with aiohttp
+==================
+
+aiohttp is used to build useful libraries built on top of it,
+and there's a page dedicated to list them: :ref:`aiohttp-3rd-party`.
```

Comment:
@tarekziade thanks for correction.
Now I finally understand why @socketpair raised the issue.
Nothing to do here.